### PR TITLE
remove transitive dependency on commons-lang

### PR DESCRIPTION
**Pull request by chrischall**
Newer versions of spring-data-elasticsearch do not include commons-lang. Rather than add it, I thought it better to remove the dependency on it.

This is my first GitHub PR (not first git PR), so apologies for any protocol violations!

### CODE REVIEW

File: spring-data-jest/src/main/java/com/github/vanroy/springdata/jest/MappingBuilder.java [hunk 1]
```diff
@@ -82,7 +81,7 @@ static XContentBuilder buildMapping(Class clazz, String indexType, String idFiel
 		// Properties
 		XContentBuilder xContentBuilder = mapping.startObject(FIELD_PROPERTIES);
 
-		mapEntity(xContentBuilder, clazz, true, idFieldName, EMPTY, false, FieldType.Auto, null);
+		mapEntity(xContentBuilder, clazz, true, idFieldName, "", false, FieldType.Auto, null);
```
Comment (chrischall) [inline spring-data-jest/src/main/java/com/github/vanroy/springdata/jest/MappingBuilder.java]: I almost made an `EMPTY` const in this class, but went with `""`.

File: spring-data-jest/src/main/java/com/github/vanroy/springdata/jest/mapper/DefaultJestResultsMapper.java [hunk 2]
```diff
@@ -139,7 +139,7 @@ public EntityMapper getEntityMapper() {
 	}
 
 	private <T> T mapEntity(String source, Class<T> clazz) {
-		if (isBlank(source)) {
+		if (!hasText(source)) {
```
Comment (chrischall) [inline spring-data-jest/src/main/java/com/github/vanroy/springdata/jest/mapper/DefaultJestResultsMapper.java]: The `!` makes is slightly less readable.

File: spring-data-jest/src/test/java/com/github/vanroy/springdata/jest/JestElasticsearchTemplateTests.java [hunk 3]
```diff
@@ -132,7 +132,7 @@ public void before() {
 	@Test
 	public void shouldReturnCountForGivenCriteriaQuery() {
 		// given
-		String documentId = randomNumeric(5);
+		String documentId = String.valueOf(ThreadLocalRandom.current().nextLong()).substring(1, 6);
```
Comment (chrischall) [inline spring-data-jest/src/test/java/com/github/vanroy/springdata/jest/JestElasticsearchTemplateTests.java]: I really struggle with this one. Spring's `StringUtils` doesn't offer anything similar, so I concocted my own. There is a slight risk of `ThreadLocalRandom.current().nextLong()` returning a value with less than 5 digits in it, but it's infinitesimally small.

**Comment by VanRoy** [comment]
@chrischall Thanks a lot for your help. It's released in 3.1.4.

**Pull request merged**